cmd/snap-confine/mount-support-nvidia.c

```diff
@@ -33,6 +33,8 @@
 #include "../libsnap-confine-private/string-utils.h"
 #include "../libsnap-confine-private/utils.h"
 
+#define SC_NVIDIA_DRIVER_VERSION_FILE "/sys/module/nvidia/version"
```

Comment:
How stable is this name? For how long is iit available?

Reply:
We are already using it and it is present in all the driver versions out there.

Reply:
Ya I just moved the existing define in the file to be available where I need it, but the name is stable for loaded kernel module.